ROMFS/px4fmu_common/init.d/13005_vtol_AAERT_quad

```diff
@@ -60,7 +60,7 @@ then
 	param set VT_IDLE_PWM_MC 1080
 	param set VT_TYPE 2
 
-	param set PWM_AUX_DIS5 950
+	param set PWM_AUX_DISARMED 950
 fi
```

Comment:
Similar to `13000_generic_vtol_standard`, this file was originally:
```
set PWM_ACHDIS 5
set PWM_AUX_DISARMED 950
```

and will require the same corrective action as `13000_generic_vtol_standard`.